login.html

```diff
@@ -55,27 +46,28 @@
   <body>
     <a href="#container" class="skip-nav">본문 바로가기</a>
     <div id="wrap">
-      <header class="header"></header>
-      <div id="container" class="login-page">
+      <div id="container" class="auth-page">
         <div class="inner03">
-          <a href="" class="login-logo">
+          <a href="/" class="auth-logo">
             <img src="/images/sub/logo_big.svg" alt="판다마켓" />
           </a>
-          <ul class="login-form">
-            <li>
-              <label for="email">이메일</label>
-              <input type="text" id="email" placeholder="이메일을 입력하세요" />
-            </li>
-            <li>
-              <label for="password">비밀번호</label>
-              <input type="password" id="password" placeholder="비밀번호를 입력하세요" />
-              <button type="button" class="toggle-password">
-                <img src="images/common/ic_eye_off.svg" alt="비밀번호 보기" />
-              </button>
-            </li>
-          </ul>
-          <button type="button" class="login-btn">로그인</button>
-          <div class="login-sns">
+          <form action="" onsubmit="login">
+            <ul class="auth-form">
+              <li>
+                <label for="email">이메일</label>
+                <input type="text" id="email" placeholder="이메일을 입력하세요" />
+              </li>
+              <li>
+                <label for="password">비밀번호</label>
+                <input type="password" id="password" placeholder="비밀번호를 입력하세요" />
+                <button type="button" class="toggle-password">
+                  <img src="images/common/ic_eye_off.svg" alt="비밀번호 보기" />
+                </button>
+              </li>
+            </ul>
+            <button type="submit" onclick="login" class="auth-btn">로그인</button>
```

Comment:
💊 제안
form 태그에 onsubmit 이벤트 발생시 login 함수를 실행하라고 작업해주셨으니 여기서는 안 적어주시는게 관리 포인트를 줄이는 측면에서 더 좋습니다~
만약 함수명이 onLogin으로 변경된다고 한다면 아래처럼 수정하신 경우 form의 onsumbit 속성만 변경해주시면 됩니다~

```suggestion
            <button type="submit" class="auth-btn">로그인</button>
```